pkg/kudoctl/cmd/install/install.go

```diff
@@ -47,24 +48,29 @@ func Run(args []string, options *Options, fs afero.Fs, settings *env.Settings) e
 	return err
 }
 
-func validate(args []string) error {
+func validate(args []string, opts *Options) error {
 	if len(args) != 1 {
 		return clog.Errorf("expecting exactly one argument - name of the package or path to install")
 	}
 
+	if opts.InCluster {
+		if opts.RepoName != "" || opts.AppVersion != "" || opts.SkipInstance {
+			return clog.Errorf("you can't use repo-name, app-version or skip-instance options when installing from in-cluster operators")
+		}
+	}
```

Comment:
If I read the code correctly, the `operator-version` parameter is *not* optional if `--in-cluster` is used. If that stays this way, we should check it here.

Reply:
Good catch. done